packages/vdev.rb

```diff
@@ -3,44 +3,42 @@
 class Vdev < Package
   description 'A device-file manager for *nix'
   homepage 'https://github.com/jcnelson/vdev'
-  version 'ceb7a6'
+  version 'ceb7a6c'
   source_url 'https://github.com/jcnelson/vdev/archive/ceb7a6c4f44dec542dc1c3c3d5abd27dec7f3e0e.tar.gz'
   source_sha256 'dbf561890aa70a8619506d166803a72d0c2a5b7590226feef784ec623bcb4739'
 
   binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/vdev-ceb7a6-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/vdev-ceb7a6-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/vdev-ceb7a6-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/vdev-ceb7a6-chromeos-x86_64.tar.xz',
+     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/vdev-ceb7a6c-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
-    aarch64: 'bf9bd2fdde42e0897e6aab47a18e2f2649b6dbd01de14bf6165c55c2a716e133',
-     armv7l: 'bf9bd2fdde42e0897e6aab47a18e2f2649b6dbd01de14bf6165c55c2a716e133',
-       i686: '8bb09676d68f9b00da89f0e9bc9499bc5d38332081cbbc698266f74644af89e1',
-     x86_64: '45e3ca2fe33ec0dce519ad5053b63fefc4c50a38ce5fb842faca3ec941006f61',
+     x86_64: '826658f93e7a9c31eda6adfc1d96f9e1ded2c8becf4bbc0a8315ba4e6a77da3f',
   })
 
-  depends_on 'glibc'
   depends_on 'fuse'
   depends_on 'libpstat'
   depends_on 'fskit'
   depends_on 'lvm2'
 
+  def self.patch
+    system "sed -i 's,-fstack-protector -fstack-protector-all ,,g' buildconf.mk libudev-compat/Makefile"
+    system "sed -i 's,attr/xattr.h,sys/xattr.h,g' fs/fs.h"
+  end
+
   def self.build
-    system "make"
-    system "make -C vdevd OS=LINUX"
-    system "make -C example"
-    system "make -C hwdb"
-    system "make -C libudev-compat"
-    system "make -C fs"
+    system "make LIBDIR=#{CREW_LIB_PREFIX}"
+    system "make LIBDIR=#{CREW_LIB_PREFIX} -C vdevd OS=LINUX"
+    system "make LIBDIR=#{CREW_LIB_PREFIX} -C example"
+    system "make LIBDIR=#{CREW_LIB_PREFIX} -C hwdb"
+    system "make LIBDIR=#{CREW_LIB_PREFIX} -C libudev-compat"
+    system "make LIBDIR=#{CREW_LIB_PREFIX} -C fs"
```

Comment:
I assume this means all the files are installed under `#{CREW_LIB_PREFIX}`?  If anything isn't, we would need `PREFIX=#{CREW_PREFIX}`, right?

Reply:
I suppose so. Let me try that rebasing URL you gave me.